src/base/bittorrent/bencoderesumedatastorage.cpp

```diff
@@ -26,27 +26,52 @@
  * exception statement from your version.
  */
 
-#include "resumedatastorage.h"
+#include "bencoderesumedatastorage.h"
 
 #include <libtorrent/bdecode.hpp>
 #include <libtorrent/bencode.hpp>
+#include <libtorrent/create_torrent.hpp>
 #include <libtorrent/entry.hpp>
 #include <libtorrent/read_resume_data.hpp>
 #include <libtorrent/write_resume_data.hpp>
 
 #include <QByteArray>
 #include <QRegularExpression>
 #include <QSaveFile>
+#include <QThread>
 
 #include "base/algorithm.h"
+#include "base/exceptions.h"
 #include "base/global.h"
 #include "base/logger.h"
 #include "base/profile.h"
 #include "base/utils/fs.h"
 #include "base/utils/io.h"
 #include "base/utils/string.h"
+#include "infohash.h"
+#include "loadtorrentparams.h"
 #include "torrentinfo.h"
 
+const char RESUME_FOLDER[] = "BT_backup";
+
+namespace BitTorrent
```

Comment:
I would put anonymous namespace before other named namespaces, but this is not a hard requirement.